pyproject.toml

```diff
@@ -1,5 +1,6 @@
 [build-system]
-requires = ["setuptools",
+# setuptools pinned due to https://github.com/astropy/astropy/issues/14025
+requires = ["setuptools<65.6.0",
```

Comment:
We don't really on `numpy.distutils` during installation, so this is not needed.